scripts/install

```diff
@@ -53,6 +53,7 @@ ynh_app_setting_set --app=$app --key=admin --value=$admin
 ynh_app_setting_set --app=$app --key=password --value=$password
 ynh_app_setting_set --app=$app --key=group_name --value="$group_name"
 ynh_app_setting_set --app=$app --key=group_description --value="$group_description"
+ynh_app_setting_set --app=$app --key=architecture --value=$architecture
```

Comment:
Eeeeh I don't think we wanna do this, 'cause the app may be backup/restored to a different architecture ? (And this setting doesn't seem to be loaded in any other script ?)

```suggestion
```